erde/lib.lua

```diff
@@ -193,8 +193,7 @@ end
 -- IMPORTANT: THIS IS AN ERDE SOURCE LOADER AND MUST ADHERE TO THE USAGE SPEC OF
 -- `__erde_internal_load_source__`!
 local function run_string(source, options)
-  local result = { __erde_internal_load_source__(source, options) }
-  return unpack(result)
+  return __erde_internal_load_source__(source, options)
```

Comment:
due to the way error rewriting works and inconsistencies with stack traces across Lua versions, I try to avoid doing tall calls to `__erde_internal_load_source__`, otherwise we actually lose some traceback information.

Not sure if its completely clear but there is a comment in `__erde_internal_load_source__` that tries to describe it better, but might be easier to see it first hand (compare error stacktraces across Lua versions when tail calling this)